### PR TITLE
Adding illustrations to academic-industry chapters

### PR DESCRIPTION
**Pull request by vhellon**
### Summary

Fixes 

Updating the academic-industry chapters with the illustrations made during the May 2023 Book Dash

### List of changes proposed in this PR (pull-request)

- [x] Images already added to the figures folder by Julien in (#3234) #3260
- [x] Figure list doesn't seem to exist any more?
- [x] Added the images to the chapters

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.

### CODE REVIEW

**Comment by vhellon** [comment]
Looks like the second image loads fine, but the first one doesn't.... unsure why as copied and pasted the figure text in each chapter

**Comment by Arielle-Bennett** [comment]
@vhellon have you spoken to any of the infrastructure folks about this? The build is failing because the .jpg is listed as "not readable" but loads fine on GitHub. Maybe we need to try deleting and re-adding it?

File: book/website/collaboration/academic-industry.md [hunk 1]
```diff
@@ -11,6 +11,17 @@
 | {ref}`Community Managers: Overview<cl-infrastructure-community-managers>` | Helpful  | Beginner | An overview of the Community Manager Role |
 | {ref}`Guide to Planning a Community<cl-new-community-guide>` | Helpful | Beginner | The Guide to Planning a Community are helpful context for the Community Building Subchapter in this section |
 
+```{figure} ../../figures/academic-industry-partnership.jpg
```
Comment (Arielle-Bennett) [inline book/website/collaboration/academic-industry.md:14]:
```suggestion
```{figure} ../../figures/academic-industry-partnership.*
```

File: book/website/collaboration/academic-industry/academic-industry-personal-story.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 (cl-academic-industry-personal-story)=  
 # Turing-Roche Community Activities Personal Story
 
+```{figure} ../../figures/turing-roche-partnership.jpg
```
Comment (Arielle-Bennett) [inline book/website/collaboration/academic-industry/academic-industry-personal-story.md:4]:
```suggestion
```{figure} ../../figures/turing-roche-partnership.*
```

**Review by Arielle-Bennett — COMMENTED** [review]
Have updated with different syntax, hopefully that helps!

**Comment by vhellon** [comment]
> Have updated with different syntax, hopefully that helps!

Thank you! What's werid was one figure was working but the other wasn't... but lets see if this fixes it

**Comment by kallewesterling** [comment]
Hello! Checked this branch and it looks like you just need to change `../../figures/academic-industry-partnership.jpg` to `../figures/academic-industry-partnership.*`. (Changed path but also removal of the definitive file ending.)

File: book/website/collaboration/academic-industry.md (hunk 1 above)
Comment (kallewesterling) [reply book/website/collaboration/academic-industry.md:14]:
```{figure} ../figures/academic-industry-partnership.jpg

**Comment by Arielle-Bennett** [comment]
@kallewesterling what on earth is this error message? 

`/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/academic-industry.md:1: WARNING: a suitable image for html builder not found: ../figures/academic-industry-partnership.*`

**Comment by kallewesterling** [comment]
It looks like you may have merged in the edits from main into this branch, @Arielle-Bennett!

image

**Review by kallewesterling — APPROVED** [review]

**Pull request merged**